wolfcrypt/src/sp_dsp32.c

```diff
@@ -4533,14 +4536,12 @@ void wc_ecc_fp_free(void)
 
 AEEResult wolfSSL_open(const char *uri, remote_handle64 *handle) 
 {
-   void *tptr;
   /* can be any value or ignored, rpc layer doesn't care
    * also ok
    * *handle = 0;
    * *handle = 0xdeadc0de;
    */
-   tptr = (void *)malloc(1);
-   *handle = (remote_handle64)tptr;
+   *handle = (remote_handle64)malloc(1);
```

Comment:
Does it make sense to check the allocation for handle succeeded?

Reply:
I'm a little confused by the purpose of the this. I think it's a method to generate a unique ID mostly for debug any value except `-1` seems like it would work. Not sure if we should fail if `malloc` fails.

Reply:
@elms, Looks like that code is used for DSP debug build. It's probably best to keep the same behavior as you have it.